src/Autodoc.zig

```diff
@@ -585,7 +585,7 @@ const DocData = struct {
             host_size: ?Expr = null,
             is_ref: bool = false,
             is_allowzero: bool = false,
-            is_mutable: bool = false,
+            is_const: bool = false,
```

Comment:
should this be `true` ?

Reply:
I didn't think so because everything else is `false` or `null`.
This default value does seem to be used here though: https://github.com/ziglang/zig/blob/7f7d5337d5c8036562aab0c20a463bc382a7a0c5/src/Autodoc.zig#L1748
Not sure if that's overwritten later.
Might change this later. Unfortunately there are some CI failures so I'm definitely missing something here.

Reply:
This is a missing piece of functionality in Autodoc. It should read the relative information from ZIR but right now it doesn't. Feel free to ignore it

Reply:
Thanks! I'll do so.